pub/css/main.css

```diff
@@ -60,27 +60,353 @@ textarea {
 /* ==========================================================================
    Author's custom styles
    ========================================================================== */
+ 
+/* TOP SECTION / HEADER */
+
+header {
+    text-align: center;
+    background-image: url("http://loneswimmer.files.wordpress.com/2012/05/low_force_3_guillamene_ne.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 10em;
+    font-family: 'Oswald', sans-serif;
+    position: relative;
+}
+
+h1 {
+    color: white;
+    margin: 0 auto;
+    text-shadow: 2px 2px #000;
+    
+}
+
+h2 {
+    font-size: 1.6ex;
+    margin: 0;
+}
+
+
+h3 {
+    max-width: 8em;
+    display: block;
+    font-size: 2.6em;
+    margin-right: 2em;
+    margin-left: 0.5ex;
+    color: #004f91;
+    font-weight: 400;
+    margin-bottom: 0;
+    font-family: 'Oswald', sans-serif;
+}
+
+.search a {
+    text-decoration: none;
+    color: black;
+    padding-right: 0.8ex;
+}
+
+.logo img {
+    height: 1em;
+    width: 1em;
+    border-radius: 50%;
+}
+
+.hidden {
+    display: none;
+}
+/* MAIN ARTICLE SECTION*/
+
+.date {
+    display: block;
+    color: #999;
+}
+
+.main {
+    width: 90%;
+    margin: 0 auto;
+    font-family: 'Open Sans', sans-serif;
+}
+
+#article-link {
+    color: #999;
+}
 
+blockquote {
+    border-left: solid #999 2px;
+    padding-left: 1em;
+    margin-left: 0;
+    font-size: 2em;
+    color: #004f91;
+    font-family: 'Oswald', sans-serif;
+}
+
+.share a {
+    height: 1.2em;
+    width: 1.2em;
+    text-decoration: none;
+    color: white;
+    background-color: #162876;
+    text-align: center;
+    line-height: 2.2ex;
+    margin: 1ex -0.1ex 1.5ex 1ex;
+    border-radius: 0.3ex;
+}
+
+.fa-share-square-o {
+    background-color: blue;
+    
+}
+
+/* SIDE COLUMN */
+
+aside {
+    text-align: center;
+    margin: 0 auto;
+    width: 23em;
+}
+.author-large {
+    height: 8em;
+    width: 8em;
+    border-radius: 50%;
+}
+
+section > h4 {
+    margin-bottom: -0.5ex;
+}
```

Comment:
Might want to use more specific selector here... In a larger project, this would target _any_ `<h4>` directly inside _any_ `<section>` tag. If you used that construct elsewhere in the site, you might get some hard-to-debug side effects.